lib/screens/item/widgets/main_view.dart

```diff
@@ -8,8 +8,10 @@ import 'package:flutter_slidable/flutter_slidable.dart';
 import 'package:hacki/blocs/blocs.dart';
 import 'package:hacki/config/constants.dart';
 import 'package:hacki/cubits/cubits.dart';
+import 'package:hacki/cubits/github/github_cubit.dart';
```

Comment:
export in `cubits/cubits.dart`.

Reply:
yes we can, I request you to do these minor changes after integrating and before releasing as I might break the underlying principle